setup/gcp/installer/forseti_client_installer.py

```diff
@@ -0,0 +1,96 @@
+# Copyright 2017 The Forseti Security Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Forseti CLI installer"""
+
+from forseti_installer import ForsetiInstaller
+
+from configs.client_config import ClientConfig
+from util.gcloud import (get_forseti_server_info,
```

Comment:
This is importing individual methods?  I don't think we should do this.  Just do ```from util import gcloud```.

Reply:
Updated.